Source/Bugsnag.m

```diff
@@ -215,12 +215,16 @@ + (void)leaveBreadcrumbForNotificationName:
     }
 }
 
-+ (void)setBreadcrumbCapacity:(NSUInteger)capacity {
++ (void)setMaxBreadcrumbs:(NSUInteger)capacity {
     if ([self bugsnagStarted]) {
         self.notifier.configuration.breadcrumbs.capacity = capacity;
```

Comment:
The default capacity is now 25 according to the spec, with bounds of 0-100 - this should be enforced.